# generated by patch-package 6.4.8 on 2022-03-28 21:29:54
#
# command:
#   npx patch-package @pnpm/lockfile-file
#
# declared package:
#   @pnpm/lockfile-file: 5.0.0
#
diff --git a/node_modules/@pnpm/lockfile-file/lib/read.js b/node_modules/@pnpm/lockfile-file/lib/read.js
index 4eff420..6f7367f 100644
--- a/node_modules/@pnpm/lockfile-file/lib/read.js
+++ b/node_modules/@pnpm/lockfile-file/lib/read.js
@@ -27,7 +27,7 @@ async function readWantedLockfileAndAutofixConflicts(pkgPath, opts) {
 }
 exports.readWantedLockfileAndAutofixConflicts = readWantedLockfileAndAutofixConflicts;
 async function readWantedLockfile(pkgPath, opts) {
-    const lockfilePath = path_1.default.join(pkgPath, constants_1.WANTED_LOCKFILE);
+    const lockfilePath = opts.lockfilePath || path_1.default.join(pkgPath, constants_1.WANTED_LOCKFILE);
     return (await _read(lockfilePath, pkgPath, opts)).lockfile;
 }
 exports.readWantedLockfile = readWantedLockfile;
